### DISCUSSION TITLE
missing guide for 'prepare to receive bitcoins etc'

### DISCUSSION THREAD

**Discussion by maphew** [theDanielJLewis/podcasting2.org#9] [Q&A]
Among other things https://podcasting2.org/podcasters says 'Prepare your podcast to receive streaming Bitcoin/Satoshis and Boostagrams through “Value 4 Value.” Here’s a guide.' which links to https://value4value.info/

On v4v there are a number of pages about what it is and nice sounding things about it's benefits. I've yet to find the advertised instructions for "how to prepare to receive ..." though. A victim of link rot?

**Comment by jamescridland** [comment]
Hello!

You’re right, we could do a better job explaining how to get yourself set up in the simplest way possible. 

That’s going to be necessarily opinionated, and point to one specific company, but I think it’s doable.

Definitely worth thinking about.

**Comment by maphew** [comment]
thanks for the quick response!
imo it's okay to use a single company or org in order to have clearer first timer docs, though add a paragraph to note this is a single example of many, with pointers. 

As a newbie it's disconcerting to be jumping from site to site to site while trying to follow a single path. It's too strongly reminiscent of the bad old days of pop-up whack-a-mole when stumbling into a gambling or anti-virus webring, where you _know_ they were trying to trick you into making a bad click.
